### ISSUE TITLE
add querystr to nchan_authorize_request  url

### ISSUE THREAD

**Issue by notedit** [slact/nchan#166]
i want to do some thing like this

```

location ~ /sub/(\w+)$ {

            nchan_channel_id $1;
            nchan_authorize_request /subauth?$query_string;
            nchan_subscriber;
            nchan_channel_group test;

        }

        location  = /subauth {

            proxy_pass http://xxxxxxx:8011/apis/subauth?$query_string;
        }

```

it seems that does not work.

**Comment by notedit** [comment]
done,   add query_string to the header

**Comment by slact** [comment]
this was actually a bug I fixed a few days ago: 93fb352ebe2 8788444dde8ee9 . You will not need to write the query string into a header after the next release.

**Comment by notedit** [comment]
is this fixed   in the https://nchan.slact.net/download/nginx-common.ubuntu.deb  ?
